bin/p2-dsctl/main.go

```diff
@@ -129,7 +131,14 @@ func main() {
 			log.Fatalf("Error occurred: %v", err)
 		}
 
-		ds, err := dsstore.Create(manifest, minHealth, name, selector, podID, *createTimeout)
+		ctx, cancelFunc := transaction.New(context.Background())
+		defer cancelFunc()
+		ds, err := dsstore.Create(ctx, manifest, minHealth, name, selector, podID, *createTimeout)
```

Comment:
Is this for the future? What's the advantage of a transaction with only one operation?